src/System.Text.Json/tests/Serialization/TestClasses.NonGenericCollections.cs

```diff
@@ -180,6 +180,11 @@ public IEnumerator GetEnumerator()
         }
     }
 
+    public class ReadOnlyWrapperForIList : WrapperForIList
+    {
+        public override bool IsReadOnly => true;
```

Comment:
What does overriding this do?

Reply:
It provides a collection to test [this path](https://github.com/dotnet/corefx/blob/de35d9326c04b2b6445baf4f2dec66b20de56b3d/src/System.Text.Json/src/System/Text/Json/Serialization/ReadStackFrame.cs#L322) which throws for readonly IList types.